Documentation/Compression.md

```diff
@@ -0,0 +1,34 @@
+# Compression
+
+## Gzip
+
+```{.cs}
+
+var myByteArray = new byte[] {1, 2, 3, 5, 6, 7 };
```

Comment:
I think that we adequately cover all we do in the readme.MD file. I would suggest you just copy-paste that in here as well.
(perhaps keeping the `myByteArray` at the top.

Reply:
But that's what I did, only the other way around, first did the Compression.md and then copied everything (except variable definitions) to the ReadMe.md file

Reply:
Ah alright!